sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -167,6 +179,7 @@ export class BatchingReceiver extends EventHubReceiver {
       };
 
       onReceiveClose = async (context: EventContext) => {
```

Comment:
I noticed `onReceiveClose` and `onSessionClose` are both `async` functions, but don't call `await`. Do they have to be functions that return a promise, or can they just be regular functions?

Edit: Realized you didn't change those functions to be async/not async, but still curious!

Reply:
Yes, they should be regular functions. Do you want to send a PR :) ?